autoprotocol/builders.py

```diff
@@ -1251,12 +1254,13 @@ class LiquidHandleBuilders(InstructionBuilders):
     """
     def __init__(self):
         super(LiquidHandleBuilders, self).__init__()
-        self.liquid_classes = ["air", "default"]
+        self.liquid_classes = ["air", "default", "viscous"]
```

Comment:
This can be a set.

Reply:
To argue in favor of backwards compatibility, this should remain a list.
While I agree a set would be useful here to limit duplicates, I do not think it's a good idea.

Reply:
Hmm... good point to both... Since this is a heavily used functions, (and also nested with bunch of subfunctions), I'm leaning toward keeping it as a List for now as well...